depends/packages/qt.mk

```diff
@@ -29,25 +27,18 @@ $(package)_config_opts += -c++std c++11
 $(package)_config_opts += -confirm-license
 $(package)_config_opts += -dbus-runtime
 $(package)_config_opts += -hostprefix $(build_prefix)
-$(package)_config_opts += -no-alsa
```

Comment:
Is the reason we can drop these lines that these options been removed from QT?

Reply:
The config_opts lines I removed are no longer supported by qt's configure.